### PR TITLE
Amls

### PR DESCRIPTION
**Pull request by XxOinvizioNxX**
Статья о проекте AMLS
Доступны 2 версии: на русском и английском

### CODE REVIEW

**Comment by okalachev** [comment]
Hi, please, see the markdownlint issues: https://travis-ci.org/github/CopterExpress/clover/jobs/763178733#L939.

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:3]: Please, commit images used in the article into the repo.

File: docs/ru/amls.md
Comment (okalachev) [inline docs/ru/amls.md:51]: Это так и задумано, что в русской статье английские заголовки?

**Comment by XxOinvizioNxX** [comment]
Как я понимаю, нам нужно исправить всё с 937 по 1147 строки?
https://travis-ci.org/github/CopterExpress/clover/jobs/763341917

**Comment by okalachev** [comment]
Да. Но попозже я могу сам что-то поисправлять, если будут затруднения.

File: docs/ru/amls.md
Comment (okalachev) [inline docs/ru/amls.md:55]: Please, insert anchors using this syntax: ```## Title {#anchor-id}```, not using html.

File: docs/ru/amls.md
Comment (okalachev) [inline docs/ru/amls.md:361]: Wouldn't it be better to embed the videos into the article using iframe? It's done like here:

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/HdHbZFz7nR0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

You also won't need preview images in the repo.

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:9]: Grappling?

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:17]: I propose to make these `mailto:` links.

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:87]: interfering?

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:109]: incoming?

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:287]: average? But I guess it's to difficult to change the code.

File: docs/en/amls.md
Comment (okalachev) [inline docs/en/amls.md:327]: control

**Review by okalachev — CHANGES_REQUESTED** [review]
Please embed the videos with iframes and resolve other issues.

**Comment by okalachev** [comment]
I performed some edits, please, give me the rights to push into your branch.

This is done with "Allow edits from maintainers" checkbox: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork.

**Comment by XxOinvizioNxX** [comment]
It will be better if we make the edits ourselves

File: docs/en/amls.md
Comment (XxOinvizioNxX) [reply docs/en/amls.md:287]: This is not quite an average. This rotating memory is used not only here. This will be replaced by classic D and I-factors in the future.

**Comment by okalachev** [comment]
These are mostly trivial edits and correct adding the article to the list of copter hack project. It's more convenient to have them in one PR, than doing them later.

If you enable this checkbox I will be able to push only to the branch you make the PR agains. I can't change another branches.

This is my full patch currently: https://gist.github.com/okalachev/4baaf0a10428d56e8db04470d92b5ca7. You can safely allow me to push it.

File: docs/ru/amls.md
Comment (XxOinvizioNxX) [reply docs/ru/amls.md:361]: Sorry, but the code with the iframe is displayed as html code and not as video. It didn't work.

Comment (okalachev) [reply docs/ru/amls.md:361]: It won't work on GitHub, but it will on the main site clover.coex.tech. The site is the main way of viewing this documentation.

File: docs/ru/amls.md
Comment (XxOinvizioNxX) [reply docs/ru/amls.md:55]: Anchors didn't work. They just displayed as code

File: docs/en/amls.md
Comment (okalachev) [reply docs/en/amls.md:287]: No, I mean the words spells average, not avarage.

File: docs/ru/amls.md
Comment (okalachev) [reply docs/ru/amls.md:55]: They will work on the main site, clover.coex.tech. See: https://clover.coex.tech/en/programming.html#positioning, https://github.com/CopterExpress/clover/blob/master/docs/en/programming.md#positioning-system-positioning.

Comment (okalachev) [reply docs/ru/amls.md:55]: And this code is not working in gitbook build (for me, at least).

**Comment by Yas-lab** [comment]
We've corrected all needed edits please check.

**Comment by XxOinvizioNxX** [comment]
Unbelievable. The build was successful...

**Review by XxOinvizioNxX — COMMENTED** [review]
All edits have been made

**Comment by okalachev** [comment]
I can't merge as there is a merge conflict. And you won't give me pushing right so I could operatively fix this.

**Comment by okalachev** [comment]
@XxOinvizioNxX, have written you in Telegram

**Pull request merged**
